scripts/workflow/push-artefact.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# fail on first error
+set -e
+
+export ARTEFACT_BUCKET_NAME="${ARTEFACT_BUCKET_NAME:-""}"
+export WORKSPACE="${WORKSPACE:-""}"
+export COMMIT_HASH="${COMMIT_HASH:-""}"
+export DIRECTORY="${DIRECTORY:-""}"
+export SERVICE="${SERVICE:-""}"
+
+# check exports have been done
+EXPORTS_SET=0
+# Check key variables have been exported - see above
+
+if [ -z "$COMMIT_HASH" ] ; then
+  echo Set COMMIT_HASH to identify which artefacts to be deployed
+  EXPORTS_SET=1
+fi
+
+if [ -z "$WORKSPACE" ] ; then
+  echo Set WORKSPACE to the workspace to action the terraform in
+  EXPORTS_SET=1
+fi
+
+if [ -z "$ARTEFACT_BUCKET_NAME" ] ; then
+  echo Set ARTEFACT BUCKET NAME to the artefact bucket name to action the terraform in
+  EXPORTS_SET=1
+fi
+
+if [ -z "$DIRECTORY" ] ; then
+  echo Set COMMIT_HASH to identify which artefacts to be deployed
```

Comment:
cut and paste error - should be DIRECTORY

Reply:
amended